src/lib/notify.js

```diff
@@ -69,11 +69,7 @@ const notify = function(metadata, config) {
       return handleNotify(metadata, config.notify);
     } else {
       // multiple notifications endpoints PR #108
-      for (const key in config.notify) {
-        if (config.notify.hasOwnProperty(key)) {
-          return handleNotify(metadata, config.notify[key]);
-        }
-      }
+      return Promise.all(_.map(config.notify, (key) => handleNotify(metadata, key)));
```

Comment:
We should catch here any error and display them with the `logger.error`

Reply:
Should I write this logic in `catch` block?

Reply:
We are already catching error. 
https://github.com/verdaccio/verdaccio/blob/dce43c06827d64a880979c5eb4684d6ee49a59d2/src/lib/notify.js#L51
It is displaying error whenever there is an error in posting a message.

Reply:
@ayusharma you are right, the error is handled there.